lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -0,0 +1,367 @@
+package Genome::Model::Tools::Transcriptome::ErccMapUnaligned;
+
+use strict;
+use warnings;
+
+use Genome;
+use Path::Class;
```

Comment:
I seem to be alone but in general I always explicitly import rather than allow default imports.